### PR TITLE
Sample code that handles messages sent from the IoT Creators platform

### PR DESCRIPTION
**Pull request by sargo**
This sample code shows how to receive a message sent from the IoT Creators platform throw the *Send a downlink message* functionality. By sending *buzzer-on* and *buzzer-off* messages users can operate a built-in buzzer.

### CODE REVIEW

File: src/main.c [hunk 1]
```diff
@@ -80,6 +85,26 @@ static void transmit_udp_data(char *data, size_t len)
 	}
 }
 
+static int receive_udp_data(char *buf, int buf_size)
+{
+	int bytes;
+	bytes = recv(client_fd, buf, buf_size, 0);
+	if (bytes < 0) {
```
Comment (Michal-Jankowski) [inline src/main.c:92]: We can simplify this cond. statement. Assuming that bytes == 0 is considered as err we can make first "if" to check against bytes <= 0 and get rid of second cond. statment in line 95.

**Review by Michal-Jankowski — COMMENTED** [review]
LGTM, +1.

File: src/main.c (hunk 1 above)
Comment (sargo) [reply src/main.c:92]: Unfortunately, `bytes == 0` is not an error, it means that the reading was successful but there is no incoming data.

Comment (Michal-Jankowski) [reply src/main.c:92]: Okay, so this is totally fine for me.

**Review by Michal-Jankowski — APPROVED** [review]

**Pull request merged**